From 5958e3ce981251e29dc51a858123ba57a97a47bd Mon Sep 17 00:00:00 2001
From: Jon TURNEY <jon.turney@dronecode.org.uk>
Date: Wed, 9 Apr 2014 21:14:07 +0100
Subject: [PATCH 17/24] Use __USE_MINGW_ANSI_STDIO for MinGW build

Use __USE_MINGW_ANSI_STDIO for ANSI C99 compatible implementation of printf()
and friends on MinGW

Signed-off-by: Jon TURNEY <jon.turney@dronecode.org.uk>
---
 Makefile.am | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Makefile.am b/Makefile.am
index 7632ba1..f1f29f3 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -69,6 +69,7 @@ endif
 if MINGW_HOST
 SOCKET_LIBS = -lws2_32
 AM_CPPFLAGS += -D_POSIX
+AM_CPPFLAGS += -D__USE_MINGW_ANSI_STDIO
 endif
 
 # Specify include paths for ac macros
-- 
2.1.1

